src/main/java/seedu/address/logic/commands/tableresults/EnrolDateTableCommandResult.java

```diff
@@ -0,0 +1,200 @@
+package seedu.address.logic.commands.tableresults;
+
+import java.util.Map;
+import java.util.Objects;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.CommandResult;
+
+/**
+ * Represents for the command result for enrol date table generation.
+ */
+public class EnrolDateTableCommandResult extends CommandResult {
+    private static final String MESSAGE_SUCCESS = "A table categorized by EnrolDate is shown";
+
+    private int janCount;
+    private int febCount;
+    private int marCount;
+    private int aprCount;
+    private int mayCount;
+    private int junCount;
+    private int julCount;
+    private int augCount;
+    private int sepCount;
+    private int octCount;
+    private int novCount;
+    private int decCount;
+
+    /**
+     * Constructor for EnrolDateTableCommandResult.
+     * @param columnValueMapping hash map instance containing column titles(String) as keys and counts(Long) as values.
+     */
+    public EnrolDateTableCommandResult(Map<String, Integer> columnValueMapping) {
+        super(MESSAGE_SUCCESS);
+        this.janCount = columnValueMapping.get("Jan");
+        this.febCount = columnValueMapping.get("Feb");
+        this.marCount = columnValueMapping.get("Mar");
+        this.aprCount = columnValueMapping.get("Apr");
+        this.mayCount = columnValueMapping.get("May");
+        this.junCount = columnValueMapping.get("Jun");
+        this.julCount = columnValueMapping.get("Jul");
+        this.augCount = columnValueMapping.get("Aug");
+        this.sepCount = columnValueMapping.get("Sep");
+        this.octCount = columnValueMapping.get("Oct");
+        this.novCount = columnValueMapping.get("Nov");
+        this.decCount = columnValueMapping.get("Dec");
```

Comment:
The months can be changed into enumerations, but due to some dependencies, can keep them for now and improve them later after the deadline.